mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -402,6 +402,35 @@ def push_boefje_task(
             task_db.status = models.TaskStatus.FAILED
             self.ctx.datastores.task_store.update_task(task_db)
 
+        # Is the ooi in other organisations? When the ooi is in other
+        # organisations we need to create tasks for those organisations as well.
+        if boefje_task.input_ooi is not None:
+            # FIXME: is the ooi shared between organisations, or is it a copy?
```

Comment:
OOI information is not shared between orga's. However, since they are the same (by primary key) most of their information is exactly the same. Small differences might still exists in properties that are present on the OOI but which are not used in the primary_key